telegram/_chat.py

```diff
@@ -2464,6 +2689,92 @@ async def forward_to(
             message_thread_id=message_thread_id,
         )
 
+    async def forwards_from(
```

Comment:
```suggestion
    async def forward_messages_from(
```

The pular form of "forward" doesn't really stick with me TBH, some for `forwards_to` 🤔

Reply:
Yeah seemed odd to me too, tagging pool because he suggested the names the last time we added forward/copy shortcuts. cc @Poolitzer

Reply:
I don't mind :D. But I think a bigger difference then a s is probably good